src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/ExpanderTests/Expander_Bindings_TemplatedControl.xaml

```diff
@@ -0,0 +1,99 @@
+﻿<UserControl
+    x:Class="UITests.Shared.Microsoft_UI_Xaml_Controls.ExpanderTests.Expander_Bindings_TemplatedControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UITests.Shared.Microsoft_UI_Xaml_Controls.ExpanderTests"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    d:DesignHeight="300"
+    d:DesignWidth="400">
+	<UserControl.Resources>
+		<ResourceDictionary>
+			<Style BasedOn="{StaticResource DefaultSettingsExpanderStyle}"
+           TargetType="local:SettingsExpander" />
+
+			<Style x:Key="DefaultSettingsExpanderStyle"
+           TargetType="local:SettingsExpander">
+				<Style.Setters>
+					<Setter Property="Background" Value="{ThemeResource SettingsCardBackground}" />
```

Comment:
SettingsCardXyz doesnt seems to exist globally in uno, nor locally here
if they are not relevant for repro, consider removing them
pretty sure you can repro with only the template property

Reply:
if you are going that route, might as well just do:
```xml
<local:SettingsExpander Header="Hello From Header in Custom Control" IsExpanded="False">
	<local:SettingsExpander.Template>
		<...>
```
and skip the page.resources section

Reply:
Kept the template in the style since there were other properties being modified that were not part of the `Template`